deploy/docker/cp-tools/research/cellprofiler-web-api/app/src/hcs_clip.py

```diff
@@ -227,9 +227,12 @@ def merge_channels(images):
         image = images[i]
         r1, g1, b1 = image.split()
         r2, g2, b2 = result.split()
-        r = np.minimum(255, np.add(np.array(r1), np.array(r2)))
-        g = np.minimum(255, np.add(np.array(g1), np.array(g2)))
-        b = np.minimum(255, np.add(np.array(b1), np.array(b2)))
+        r = np.add(np.array(r1, dtype='uint16'), np.array(r2, dtype='uint16'))
+        r = np.where(r <= 255, r, 255).astype('uint8')
```

Comment:
It looks like this lines can be extracted to a new method and reused below to avoid code duplication. Could you please check it and fix it if it possible?